assets/src/infrastructure/ui/display/Alert.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Alert as ChakraAlert, AlertIcon, AlertTitle, AlertDescription, AlertProps } from '@chakra-ui/core';
+
+interface Props extends AlertProps {
+	description: string;
+	title: string;
+}
+
+const Alert: React.FC<Props> = ({ description, title, ...props }) => (
+	<ChakraAlert
+		{...props}
+		variant='subtle'
+		flexDirection='column'
+		justifyContent='center'
+		textAlign='center'
+		height='200px'
+	>
```

Comment:
status is passed from the consumer:

![image](https://user-images.githubusercontent.com/1477453/78936700-c0c59700-7ab7-11ea-9cac-747d4af5b15a.png)

Reply:
noice 👍